src/App.css

```diff
@@ -66,6 +67,15 @@ nav {
   width: 80%
 }
 
+.heart-emoji {
+  font-size: 1.3rem;
+  color: red;
```

Comment:
This colour may not be coming through, your vote icon is currently grey with white text and extremely hard to see